<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basic_type_conversion_form</class>
 <widget class="QWidget" name="basic_type_conversion_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基本类型转换</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="endianness_label">
     <property name="text">
      <string>字节序：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="endianness_combo_box"/>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="add_type_push_button">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QSpinBox" name="count_spin_box">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QComboBox" name="type_combo_box">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="type_label">
     <property name="text">
      <string>类型：</string>
     </property>
    </widget>
   </item>
   <item row="11" column="10">
    <widget class="QCheckBox" name="auto_write_check_box">
     <property name="text">
      <string>自动</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6" colspan="2">
    <widget class="QLabel" name="to_string_method_label">
     <property name="text">
      <string>字符串格式：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="8">
    <widget class="QComboBox" name="to_string_method_combo_box">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="sizeof_label">
     <property name="text">
      <string>结构长度：</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="sizeof_val_label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="9" rowspan="3" colspan="2">
    <widget class="QPlainTextEdit" name="context_plain_text_edit"/>
   </item>
   <item row="11" column="9">
    <widget class="QPushButton" name="write_push_button">
     <property name="text">
      <string>写入</string>
     </property>
    </widget>
   </item>
   <item row="11" column="8">
    <widget class="QCheckBox" name="auto_read_check_box">
     <property name="text">
      <string>自动</string>
     </property>
    </widget>
   </item>
   <item row="11" column="6">
    <widget class="QPushButton" name="read_push_button">
     <property name="text">
      <string>读取</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" rowspan="3" colspan="4">
    <widget class="QPlainTextEdit" name="memory_plain_text_edit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="9">
    <widget class="QLabel" name="bytes_length_label">
     <property name="text">
      <string>数据长度：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="10">
    <widget class="QLabel" name="bytes_length_val_label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" rowspan="6" colspan="2">
    <widget class="QListView" name="type_list_view">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
